modules/brightMountainMediaBidAdapter.js

```diff
@@ -42,13 +55,32 @@ export const spec = {
     }
     for (let i = 0; i < validBidRequests.length; i++) {
       let bid = validBidRequests[i];
-      let traff = bid.params.traffic || BANNER
       let placement = {
         placementId: bid.params.placement_id,
         bidId: bid.bidId,
-        sizes: bid.mediaTypes[traff].sizes,
-        traffic: traff
       };
+
+      if (bid.mediaTypes.hasOwnProperty(BANNER)) {
+        placement['traffic'] = BANNER;
+        placement['sizes'] = bid.mediaTypes.banner.sizes;
+      }
+
+      if (bid.mediaTypes.hasOwnProperty(VIDEO)) {
+        placement['traffic'] = VIDEO;
+        placement['context'] = bid.mediaTypes.video.context;
+        placement['sizes'] = bid.mediaTypes.video.playerSize;
+        placement['mimes'] = bid.mediaTypes.video.mimes ? bid.mediaTypes.video.mimes : videoExt;
+        if (bid.params.video) {
+          if (bid.params.video.skippable) {
```

Comment:
Your last commit just deleted the example of how to set this,  but didn't add support for reading this from the ad unit

Reply:
Please have a look.

Reply:
@patmmccann can you check if this issue is resolved?

Reply:
Should be if (bid.mediaTypes.video.mimes), if (bid.mediaTypes.video.playback_method) etc